container/build_tar.py

```diff
@@ -375,6 +405,8 @@ def file_attributes(filename):
       output.add_empty_file(f, **file_attributes(f))
     for f in FLAGS.empty_dir:
       output.add_empty_dir(f, **file_attributes(f))
+    for f in FLAGS.empty_root_dir:
```

Comment:
In Windows there are two root directories ('Files', 'Hives') instead of one as with Linux images ('.').  These changes are actually brought over from the bazel repo where build_tar.py was forked from at some point.

In the context of the bazel repo build_tar.py isn't necessarily Docker specific so following the existing pattern of empty_dir and allowing multiple empty root directories makes sense.

In this repo I could modify this so it only adds the empty root directory needed ('Hives' since 'Files' will store all the data that Linux images normally store in '.').  If you'd like me to constrain this to 'Hives' I can.  Please let me know what approach you'd prefer.

Reply:
I like it how it is right now. Do not know enough about windows filesystem and your comments are very helpful. Your other changes to docs so far look good, with the additional context its clear this should allow multiple empty_root_dir (also, like this is generic as opposed to hardcoded for windows).